docs/user-guide/basic-usage.md

```diff
@@ -90,7 +86,7 @@ configuration.
 By default, the logback file is loaded from *$STYX_HOME/conf/logback.xml*. You can specify an alternative
 logging configuration file using {-l} or {--logback} command line options:
```

Comment:
There are some old formatting isssues with the page, such as `-l` and `—logback` above. Please address them too now as you are changing this file. Need to have a consistent formatting for this and the other pages.